ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-arm.yaml

```diff
@@ -74,6 +76,7 @@ tests:
       COMPUTE_NODE_REPLICAS: "24"
       COMPUTE_NODE_TYPE: m6g.xlarge
       CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
+      EXTRA_FLAGS: --churn-duration=20m
```

Comment:
node-density-heavy .. --profile-type=reporting not churn-duration=20m

Reply:
fixed